demos/twa-webview-fallback/src/main/java/com/google/browser/examples/twawebviewfallback/WebViewFallbackActivity.java

```diff
@@ -30,15 +31,30 @@
 public class WebViewFallbackActivity extends AppCompatActivity {
     public static final String KEY_LAUNCH_URI =
             "com.google.browser.examples.twawebviewfallback.WebViewFallbackActivity.KEY_LAUNCH_URL";
+    public static final String NAVIGATION_BAR_COLOR =
+            "com.google.browser.examples.twawebviewfallback.WebViewFallbackActivity" +
```

Comment:
Since you're concatenating strings anyway, why not do:

    private static final String KEY_PREFIX =
        "com.google.browser.examples.twawebviewfallback.WebViewFallbackActivity.";
    private static final String KEY_LAUNCH_URI = KEY_PREFIX + "KEY_LAUNCH";
    private static final String KEY_NAVIGATION_BAR_COLOR = KEY_PREFIX + "NAVIGATION_BAR_COLOR";

Note, also:

* I made them private according to my comment in `MyLauncherActivity`.
* `NAVIGATION_BAR_COLOR` gets `KEY_` as a prefix (like `KEY_LAUNCH_URI`).
* The *value* of `KEY_LAUNCH_URI` doesn't contain `KEY_` (since that seems a bit odd).

Reply:
done